src/main/PresentationExchange.ts

```diff
@@ -1,21 +1,13 @@
-import {
-  EvaluationResults,
-  IPresentationDefinition,
-  KeyEncoding,
-  PEX,
-  PresentationSignCallBackParams,
-  PresentationSignOptions,
-  SelectResults,
-  Status,
-} from '@sphereon/pex';
+import { EvaluationResults, IPresentationDefinition, KeyEncoding, PEX, PresentationSignOptions, SelectResults, Status } from '@sphereon/pex';
```

Comment:
Github gives these: 

> Argument of type 'PresentationSignCallback' is not assignable to parameter of type '(callBackParams: PresentationSignCallBackParams) => Promise<IVerifiablePresentation>'.

Reply:
yeah, as I said it's a local build of pex, I didn't push the local pex library that I'm using